content/02.main-text.md

```diff
@@ -24,10 +24,10 @@ In total, the Deep Review attracted {{deep_review_authors}} authors who were not
 
 Writing review articles in a public forum allows review authors to engage with the original researchers to clarify their methods and results and present them accurately, as exemplified [here](https://github.com/greenelab/deep-review/issues/213).
 Additionally, discussing manuscripts in the open generates valuable post-publication peer review [@pmid:25851694; @doi:10.1371/journal.pmed.1001772; @doi:10.3389/fncom.2012.00063] or pre-publication peer review of preprints [@tag:Avasthi2018_preprints].
-Because incentives are commonly lacking to provide public peer review of existing literature [@doi:10.1629/uksg.245], open collaborative reviews --- where authorship is open to anyone who makes a valid contribution --- could help spur more post-publication peer review.
+Because incentives to provide public peer review of existing literature [@doi:10.1629/uksg.245] are lacking, open collaborative reviews --- where authorship is open to anyone who makes a valid contribution --- could help spur more post-publication peer review.
 However, inviting wide authorship brings many technical and social challenges such as how to fairly distribute credit, coordinate the scientific content, and collaboratively manage extensive reference lists.
 
-To address these challenges, we developed a manuscript writing process using the Markdown language, the GitHub platform, and our new Manubot tool for automating manuscript generation.
+To address these challenges, we developed a manuscript writing process using the Markdown language, the GitHub platform, and our new [Manubot tool](https://github.com/greenelab/manubot) for automating manuscript generation.
```

Comment:
You can remove the link here. I moved it elsewhere in #92 (to `Manubot is a system for writing scholarly manuscripts via GitHub that is built upon our Python package of the same name.`).